master/nightly_llvmsvn.py

```diff
@@ -49,6 +49,12 @@
     	haltOnFailure = True
     ),
 
+    # Print versioninfo. This is particularly useful for llvm-svn build
+    # since it includes the llvm version number.
+    ShellCommand(
+    	name="versioninfo()",
+    	command=["usr/bin/julia", "-f", "-e", "versioninfo()"]
```

Comment:
Ahh, good catch, I copied over the next block and didn't realize the difference .................